.github/workflows/push.yml

```diff
@@ -0,0 +1,27 @@
+name: PR Test
```

Comment:
Name the file `build.yml` for consistency? 

https://github.com/StackExchange/dnscontrol/blob/master/.github/workflows/build.yml

Reply:
This is the name of the file we use in the rest of the work I've done.  I also have a release.yml that will be in a future PR and is used for cutting releases.